lingua_franca/res/text/en-us/normalize.json

```diff
@@ -99,7 +99,78 @@
     "you'd've": "you would have",
     "you'll": "you will",
     "you're": "you are",
-    "you've": "you have"
+    "you've": "you have",
+    "I'm'a": "I am about to",
+    "I'm'o": "I am going to",
+    "I'll've": "I will have",
+    "I'd've": "I would have",
+    "Whatcha": "What are you",
+    "amn't": "am not",
+    "'cause": "because",
+    "can't've": "cannot have",
+    "couldn't've": "could not have",
+    "daren't": "dare not",
+    "daresn't": "dare not",
+    "dasn't": "dare not",
```

Comment:
Haven't seen these ever, though could just be vernacular I'm not familiar with

Reply:
I know "daren't" but not with a S